AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -23,6 +23,8 @@
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
 ||st.work.ua/js/app-banner-
+jp.pinkoi.com###g-app-bar
+pinkoi.com#$#.s-app-banner div#m-mainview { padding-top: 51px!important; }
```

Comment:
@Yuki2718 @SKEIDs in this case, these rules have to be written via `#$#` syntax.
Btw, they are only needed on the mobile platforms.
```
pinkoi.com#$##g-app-bar { display: none !important; }
pinkoi.com#$#.s-app-banner div#m-mainview { padding-top: 51px !important; }
```

Reply:
@zloyden I think I've learned a lot. Thank you.

Reply:
You are welcome.

Reply:
@zloyden I think in this case the hidding rule should be left as it was for iOS 14 or under, because the only job the css rule does is removing placeholder. So the original rule @SKEIDs made is fine by itself without my addition. If it causes scroll locked or ugly whitespace (often seen between the top and navigation bar), sure the rules should be type-alike.

Reply:
> ugly whitespace (often seen between the top and navigation bar), sure the rules should be type-alike

@Yuki2718 this one. In such cases we decided to make rules like you said type-alike.